client/my-sites/domains/index.js

```diff
@@ -318,6 +318,13 @@ export default function () {
 		clientRender
 	);
 
+	page(
+		'/domains/domain/:domain',
```

Comment:
What do you think about using `/domains/manage/:domain/edit` here instead?  This would mean that it's exactly like the existing domain manage edit page, but without the site path part.

Reply:
You're totally right, I updated the url 👍